imports/ui/components/comments/Comments.js

```diff
@@ -14,12 +14,20 @@ export default class Comments extends Component {
   getAuthorName(authorId) {
     return Person.findOne({userId: authorId});
   }
+  // render() {
+  //   const style = this.props.comments ? 'collection chat' : '111';
+  // }
+  componentDidMount() {
+    if (!this.props.comments.length - 1) {
```

Comment:
look at example on SwitchLocaleLinks.jsx
something like 
return classNames('comments', {
      'collection chat': this.props.comments.length <= 1,
    });

should work, do not forget to import classnames package on this page